.github/helpers/generate-release-notes.sh

```diff
@@ -6,7 +6,7 @@ CHANGELOG=$(git --no-pager log --no-notes --no-decorate --oneline  v${1}...HEAD)
 ## Gather Framework version
 PLAYWRIGHT_VER=$(< package-lock.json jq -r '.dependencies["playwright"].version')
 PLAYWRIGHT_TEST_VER=$(< package-lock.json jq -r '.dependencies["@playwright/test"].version')
-NODEJS_VER=$(grep NODE_VERSION= Dockerfile | cut -d '=' -f 2)
+NODEJS_VER=$(grep NODE_VERSION: .github/workflows/release.yml | cut -d ':' -f 2 | tr -d ' ')
```

Comment:
Would reading from the `.nvmrc` file make this more robust?

```suggestion
NODEJS_VER=$(cat .nvmrc | tr -d 'v')
```